src/Utils/AST.php

```diff
@@ -118,6 +118,14 @@ public static function fromArray(array $node) : Node
         return $instance;
     }
 
+    /**
+     * @param mixed[] $node
```

Comment:
Since this method is used as a type guard to check if whatever is passed in is an AST, I think that any type should be fine.

```suggestion
     * @param mixed $maybeNode
```